### ISSUE TITLE
Has multiple elements referenced with ARIA with the same id attribute.

### ISSUE THREAD

**Issue by metakolor** [jayfreestone/priority-plus#22]
It would be awesome if the cloned and hidden nav would be stripped of its aria-labels or id's to ensure every id attribute value used in ARIA and in labels is unique.

**Comment by jayfreestone** [comment]
Yeah not a bad idea. I wonder if the ARIA labels _might_ be fine to leave because the hidden nav is wrapped in `aria-hidden` (would need to investigate, but maybe?).

The IDs I can definitely see being a problem though.
